cmd/trigger_ui/main.go

```diff
@@ -138,5 +165,5 @@ func main() {
 	router.GET("/vx-export", server.triggerHandlerGET)
 	router.POST("/vx-export", server.triggerHandlerPOST)
 
-	router.Run(":8080")
+	router.Run(os.Getenv("PORT"))
```

Comment:
I would probably add a fallback to 8080 or some other unique port if env isn't set, although it isn't critical.
```suggestion
	port := os.Getenv("PORT")
	if port == "" {
		port = "8083"
	}
	router.Run(fmt.Sprintf(":%s", port))
```